html-templates/subtemplates/blog.tpl

```diff
@@ -0,0 +1,78 @@
+{load_templates "subtemplates/people.tpl"}
```

Comment:
@BillClinton PR this revised file all the way up to emergence-skeleton, it's just adding a new option that is off by default so that's no reason to override the whole thing in this project

Reply:
done. See this PR: https://github.com/JarvusInnovations/emergence-skeleton/pull/326